### ISSUE TITLE
DLA32 1x performance

### ISSUE THREAD

**Issue by twangnh** [xingyizhou/CenterNet#270]
Hi, Xingyi, I havent change the code and I trained `dla32` with 
`python main.py ctdet --exp_id coco_dla_1x --batch_size 128 --master_batch 9 --lr 5e-4 --gpus 0,1,2,3,4,5,6,7 --num_workers 16`
however it is strange I was only able to get 34.4 mAP for 140 epoch training, do you have any clue on this?

**Comment by xingyizhou** [comment]
Hi, please make sure you are using pytorch 0.4.1 and have disabled cudnn batchnorm as in readme. We also observed a slight performance drop (lowest 35.6 without flip) in pytorch 1.0+ with or without disabling cudnn, if not change the code.

**Comment by fenglv12345** [comment]
@twangnh hi,have you solved that problem,and how?thank you very much!

**Comment by Etoye** [comment]
Same here! WIth DLA-34 1x I could achieve only 34.4 mAP with 140 epoch training.

**Comment by Etoye** [comment]
> Hi, Xingyi, I havent change the code and I trained `dla32` with
> `python main.py ctdet --exp_id coco_dla_1x --batch_size 128 --master_batch 9 --lr 5e-4 --gpus 0,1,2,3,4,5,6,7 --num_workers 16`
> however it is strange I was only able to get 34.4 mAP for 140 epoch training, do you have any clue on this?

Hello, any progress on it? Tried to train a DLA-34 1x, 2x and results right near yours, 34.6 and 34.7, with 0.4.1 torch version and fixed (turned off) batch norm. It's quite interesting how to push it higher to the original paper results.
